doc/nrf/releases_and_maturity/releases/release-notes-changelog.rst

```diff
@@ -93,9 +93,12 @@ Matter
 ------
 
 * Updated the page about :ref:`ug_matter_device_low_power_configuration` with the information about Intermittently Connected Devices (ICD) configuration.
-* Added a Kconfig option for disabling or enabling :ref:`ug_matter_configuring_read_client`.
-* Added support for PSA Crypto API for devices that use Matter over Thread.
-  It is enabled by default and can be disabled by setting the :kconfig:option:`CONFIG_CHIP_CRYPTO_PSA` Kconfig option to ``n``.
+* Added:
+  * Kconfig option for disabling or enabling :ref:`ug_matter_configuring_read_client`.
```

Comment:
```suggestion
* Added:

  * Kconfig option for disabling or enabling :ref:`ug_matter_configuring_read_client`.
```
Need the empty line here since it's a list inside a list.